service/controller/resource/tccpn/create.go

```diff
@@ -122,16 +123,19 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			return microerror.Maskf(executionFailedError, "expected one stack, got %d", len(o.Stacks))
 
 		} else if *o.Stacks[0].StackStatus == cloudformation.StackStatusCreateFailed {
-			return microerror.Maskf(executionFailedError, "expected successful status, got %#q", *o.Stacks[0].StackStatus)
-
-		} else if *o.Stacks[0].StackStatus == cloudformation.StackStatusCreateInProgress {
-			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("the tenant cluster's control plane nodes cloud formation stack has stack status %#q", cloudformation.StackStatusCreateInProgress))
-			r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
-			return nil
-		} else if *o.Stacks[0].StackStatus == cloudformation.StackStatusUpdateInProgress {
-			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("the tenant cluster's control plane nodes cloud formation stack has stack status %#q", cloudformation.StackStatusUpdateInProgress))
+			return microerror.Maskf(eventCFCreateError, "expected successful status, got %#q", *o.Stacks[0].StackStatus)
+		} else if *o.Stacks[0].StackStatus == cloudformation.StackStatusRollbackFailed {
+			return microerror.Maskf(eventCFRollbackError, "expected successful status, got %#q", *o.Stacks[0].StackStatus)
+		} else if *o.Stacks[0].StackStatus == cloudformation.StackStatusUpdateRollbackFailed {
+			return microerror.Maskf(eventCFUpdateRollbackError, "expected successful status, got %#q", *o.Stacks[0].StackStatus)
+
+		} else if stackInProgress(*o.Stacks[0].StackStatus) {
```

Comment:
simplified it a bit. 

all stack status which are in progress should be emitted via `CFInProgess`
and
all stack status which are completed should be emitted via `CFCompleted`

so that we don't have to catch all status (Create,Update,Rollback...)